docs/develop/api-reference/auth-tokens.md

```diff
@@ -0,0 +1,181 @@
+---
+sidebar_position: 7
+title: Access control token API reference information
```

Comment:
@danielamiao - are you ok with using Access Control Tokens as the main term? cc @rishtigupta and @bruuuuuuuce for the console wording

Reply:
Fine Grained Access Control (FGAC) is better